api/config/config.go

```diff
@@ -323,6 +323,8 @@ type StandardTransformerConfig struct {
 	Jaeger             JaegerConfig          `validate:"required"`
 	SimulationFeast    SimulationFeastConfig `validate:"required"`
 	Kafka              KafkaConfig           `validate:"required"`
+	// Simulator configs
+	SimulatorFeastClientMaxConcurrentRequests int `validate:"required" default:"100"`
```

Comment:
just curious, if we don't set the value, will it call the validate first or populate the default first?

Reply:
It will call default first 😅 (so actually there's no real need for `validate`)

Reply:
Unless somehow someone tries to set it as `nil` in between when `default` is set and when `validate` is called.

`default` is set over here: https://github.com/caraml-dev/merlin/blob/main/api/cmd/api/main.go#L84, via `Load`, which calls `reflectViperConfig`, which then sets all the defaults with the `default` tag.

`validate` is only called in that same file I shared, at line 100, [`reflectViperConfig](https://github.com/caraml-dev/merlin/blob/main/api/cmd/api/main.go#L100.